src/System.Memory/ref/System.Memory.cs

```diff
@@ -195,6 +200,7 @@ public static partial class MemoryExtensions
     public readonly ref partial struct Span<T>
     {
         private readonly object _dummy;
+        private readonly int _dummyInt;
```

Comment:
**Question:** When should the dummy field by read-only?

Reply:
Yes it has to be because the struct is readonly.